src/game_api/entity.hpp

```diff
@@ -361,29 +363,23 @@ struct Inventory
 {
     uint32_t money;
     uint8_t bombs;
+    /// To access/edit anything below use `ON.PRE_LEVEL_GENERATION`, those are used to transfer information to new player entity
     uint8_t ropes;
-    /// Used in level transition to transfer to new player entity, is wrong during the level

```

Comment:
Maybe keep a `see "ropes" for correct usage`

Reply:
wdym?

Reply:
all of those comments say the same thing, that's way i made it into one comment that's say "hey, from this point onward all the vars are level transition use only", and it's added "to the rope" var so it displays under it, so anything below the rope

Reply:
Here my point is also about clarity of docs. Putting a comment on one thing is bound to confuse people. If I go into the docs and just check `Inventory::health` then I don't know what the usage of it is because I would have to look at `Inventory::ropes`. To clarify that you either duplicate the comment on `Inventory::ropes` for each relevant member or you mention that same restrictions apply for each member. Having duplication in docs is not bad in a case like this, removing the duplication is harmful instead.
This one I have strong feelings about  ;)